src/entities/client.ts

```diff
@@ -0,0 +1,37 @@
+import {
+    Entity,
+    PrimaryGeneratedColumn,
+    Column,
+    CreateDateColumn,
+    UpdateDateColumn,
+    OneToMany,
+} from "typeorm";
+import { Sample } from "@/entities"
+
+@Entity()
+export class Client {
+    @PrimaryGeneratedColumn()
+    id!: number;
+
+    @Column()
+    fullName!: string;
```

Comment:
again no. I'm sure the db column will b full_name on postgres. so there has to be a column name here. if you dont provide a column name, typeorm will query fullName instead of full_name